src/simulation/dynamics/HingedRigidBodies/hingedRigidBodyStateEffector.h

```diff
@@ -58,56 +58,67 @@ class HingedRigidBodyStateEffector : public StateEffector, public SysModel {
 
 private:
     static uint64_t effectorID;        //!< [] ID number of this panel
-    double theta;                    //!< [rad] hinged rigid body angle
-    double thetaDot;                 //!< [rad/s] hinged rigid body angle rate
-    double cTheta;                  //!< -- term needed for back substitution
     double u;                        //!< [N-m] optional motor torque
-    Eigen::Vector3d r_HP_P;          //!< [m] vector pointing from primary body frame P origin to Hinge location.  If a single spacecraft body is modeled than P is the same as B
-    Eigen::Matrix3d dcm_HP;          //!< -- DCM from primary body frame to hinge frame
+
+    // Terms needed for back substitution
     Eigen::Vector3d aTheta;         //!< -- term needed for back substitution
     Eigen::Vector3d bTheta;         //!< -- term needed for back substitution
-    Eigen::Matrix3d rTilde_HP_P;     //!< -- Tilde matrix of rHB_B
-    Eigen::Matrix3d dcm_SH;          //!< -- DCM from hinge to hinged rigid body frame, S
-    Eigen::Matrix3d dcm_SP;          //!< -- DCM from body to S frame
-    Eigen::Vector3d omega_PN_S;       //!< [rad/s] omega_BN in S frame components
+    double cTheta;                  //!< -- term needed for back substitution
+
+    // Vector quantities
+    Eigen::Vector3d r_HP_P;          //!< [m] vector pointing from primary body frame P origin to Hinge location.  If a single spacecraft body is modeled than P is the same as B
+    Eigen::Vector3d omega_PN_S;      //!< [rad/s] omega_BN in S frame components
     Eigen::Vector3d sHat1_P;         //!< -- unit direction vector for the first axis of the S frame
     Eigen::Vector3d sHat2_P;         //!< -- unit direction vector for the second axis of the S frame
     Eigen::Vector3d sHat3_P;         //!< -- unit direction vector for the third axis of the S frame
     Eigen::Vector3d r_SP_P;          //!< -- Vector pointing from B to CoM of hinged rigid body in B frame components
-    Eigen::Matrix3d rTilde_SP_P;     //!< -- Tilde matrix of rSB_B
     Eigen::Vector3d rPrime_SP_P;     //!< [m/s] Body time derivative of rSB_B
-    Eigen::Matrix3d rPrimeTilde_SP_P;  //!< -- Tilde matrix of rPrime_SB_B
-    Eigen::Matrix3d ISPrimePntS_P;  //!< [kg-m^2/s] time body derivative IPntS in body frame components
     Eigen::Vector3d omegaLoc_PN_P;  //!< [rad/s] local copy of omegaBN
+    Eigen::Vector3d omega_BN_B{0.0, 0.0, 0.0};  //!< Hub/Inertial angular velocity vector in B frame components
+    Eigen::MRPd sigma_BN{0.0, 0.0, 0.0};       //!< Hub/Inertial attitude represented by MRP
```

Comment:
MRP is not a vector quantity.  Should be moved to next section listing matrices, including the DCMs.